### PR TITLE
Task1 part2 implement design

### PR DESCRIPTION
**Pull request by abood454**
part 2

### CODE REVIEW

File: task_1/style.css [hunk 1]
```diff
@@ -8,6 +8,143 @@
...
+}
+
+.coffee-item {
+    background: linear-gradient(180deg, #ECF8F1 25%, white 75%);
+    border-radius: 10px;
+    height: 450px;
+    margin: auto;
+    width: 217px;
+}
+
```
Comment (ashrafabuodeh) [inline task_1/style.css:126]: remove space

**Review by ashrafabuodeh — CHANGES_REQUESTED** [review]

File: task_1/index.html [hunk 2]
```diff
@@ -12,34 +12,103 @@
...
+                <li id="image1" onclick="MainEvent('image1')"><img class="image-size2" src="./Assets/images/coffee-icon-png-13667.png"></li>
+                <li id="Home" onclick="MainEvent('Home')">Home</li>
+                <li id="AboutUs" onclick="MainEvent('AboutUs')">About Us</li>
+                <li id="OurLands" onclick="MainEvent('OurLands')">Our Lands</li>
+                <li id="OurSuppliers" onclick="MainEvent('OurSuppliers')">Our Suppliers</li>
+                <li id="Products" onclick="MainEvent('Products')">Products</li>
+                <li id="ContactUs" onclick="MainEvent('ContactUs')">Contact Us</li>
+                <li id="image2" onclick="MainEvent('image2')"><img class="image-size" src="./Assets/images/profile Image.png"></li>
+                <li id="image3" onclick="MainEvent('image3')"><img class="image-size3" src="./Assets/images/cart-icon-28351.png"></li>
+
```
Comment (Mohsen-Awad) [inline task_1/index.html:25]: remove space

File: task_1/index.html [hunk 3]
```diff
@@ -12,34 +12,103 @@
...
+                    </div>
+
+                    <div class="coffee-item">
+                        <img class="image" src="./Assets/images/MOcha Late 02.png">
+                        <p class="coffie-title">Mocha Late</p>
+                        <p class="Des">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                    </div>
+                    <div class="More-item">
+                        <img src="./Assets/images/image 2.png" style="margin-top: 30px;">
+                        <p style="margin: 0;"> View More</p>
```
Comment (Mohsen-Awad) [inline task_1/index.html:64]: put the style in css file

File: task_1/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+     var arr = document.getElementsByTagName("li");
+
+     for (var i = 0; i < arr.length; i++)
+         arr[i].classList.remove("active");
+
+     if (!obj.className.split(' ').includes("active"))
+         obj.className += " active";
+     else
+         obj.classList.remove("active");
+ }
```
Comment (Mohsen-Awad) [inline task_1/index.js:12]: add new line at the end

File: task_1/style.css [hunk 5]
```diff
@@ -8,6 +8,143 @@
...
+.wrapper {
+    background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ECF8F1 6.25%);
+    margin: 0 auto;
+    max-width: 1440px;
+}
+
+#NavList {
+    list-style: none;
+    margin-right: 40px;
+    margin-top: 20px;
```
Comment (Mohsen-Awad) [inline task_1/style.css:20]: use shorthand margin

File: task_1/style.css [hunk 6]
```diff
@@ -8,6 +8,143 @@
...
+.image-size {
+    margin-bottom: 40px;
+    max-height: 85px;
+    max-width: 75px;
+    position: relative;
+}
+
+.image-size2 {
+    margin: 0;
+    margin-top: 5px;
```
Comment (Mohsen-Awad) [inline task_1/style.css:45]: use shorthand margin

File: task_1/style.css [hunk 7]
```diff
@@ -8,6 +8,143 @@
...
+    max-height: 60px;
+    max-width: 60px;
+}
+
+.image-size3 {
+    float: right;
+    margin: 0;
+    margin-top: 10px;
```
Comment (Mohsen-Awad) [inline task_1/style.css:53]: also use shorthand margin

File: task_1/style.css [hunk 8]
```diff
@@ -16,54 +153,49 @@
...
     display: flex;
     padding-bottom: 32px;
 }
 
 .hero-image {
-    border: 1px solid black;
-    min-width: 320px;
+    margin-left: 80px;
+    margin-right: 40px;
+    margin-top: 50px;
```
Comment (Mohsen-Awad) [inline task_1/style.css:176]: use shorthand margin

**Review by Mohsen-Awad — CHANGES_REQUESTED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.html [hunk 9]
```diff
@@ -12,34 +12,103 @@
...
+                <li id="image3" onclick="MainEvent('image3')"><img class="image-size3" src="./Assets/images/cart-icon-28351.png"></li>
+
+            </ul>
+
         </div>
         <div class="main">
-            <div class="hero-image"></div>
+            <div class="hero-image">
+                <div class="Card">
+                    <img class="image" src="Assets/images/MOCHA LATE.png">
```
Comment (laithyounes) [inline task_1/index.html:32]: change image name/title this will not run as expected on some OS/browsers

File: task_1/index.html (hunk 3 above)
Comment (laithyounes) [reply task_1/index.html:64]: yes, let's not use inline styles

File: task_1/index.html [hunk 10]
```diff
@@ -12,34 +12,103 @@
...
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
+            <div class="coffee-item">
+                <img class="image" src="./Assets/images/MOcha Late 02.png">
+                <p class="coffie-title">Mocha Late</p>
+                <p class="Des">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
```
Comment (laithyounes) [inline task_1/index.html:73]: class naming convention should be fixed here

File: task_1/index.html [hunk 11]
```diff
@@ -12,34 +12,103 @@
...
+            </div>
+            <div class="coffee-item">
+                <img class="image" src="./Assets/images/MOcha Late 02.png">
+                <p class="coffie-title">Mocha Late</p>
+                <p class="Des">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+            </div>
+            <div class="coffee-item">
+                <img class="image" src="./Assets/images/MOcha Late 02.png">
```
Comment (laithyounes) [inline task_1/index.html:81]: nit: let's change the class name here, image is too generic

File: task_1/index.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
+ const MainEvent = (ID) => {
```
Comment (laithyounes) [inline task_1/index.js:1]: please use camelCase naming convention here

File: task_1/style.css [hunk 13]
```diff
@@ -8,6 +8,143 @@
     src: url(./Fonts/Product_Sans/Product\ Sans\ Regular.ttf);
 }
 
+.wrapper {
+    background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ECF8F1 6.25%);
+    margin: 0 auto;
+    max-width: 1440px;
+}
+
+#NavList {
```
Comment (laithyounes) [inline task_1/style.css:17]: nit: we can rely more on classes

File: task_1/index.html [hunk 14]
```diff
@@ -12,34 +12,103 @@
...
+
+            </ul>
+
         </div>
         <div class="main">
-            <div class="hero-image"></div>
+            <div class="hero-image">
+                <div class="Card">
+                    <img class="image" src="Assets/images/MOCHA LATE.png">
+                    <p class="coffie-title">Mocha Late</p>
```
Comment (laithyounes) [inline task_1/index.html:33]: `coffie-title` typo

**Review by laithyounes — CHANGES_REQUESTED** [review]

**Review by laithyounes — APPROVED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.html [hunk 15]
```diff
@@ -1,45 +1,110 @@
...
         <div class="header">
+            <ul class="nav-list">
+                <li id="store-image" onclick="onclick_navbar_item('store-image')"><img class="store-size" src="./Assets/images/coffee-icon-png-13667.png"></li>
+                <li id="home" onclick="onclick_navbar_item('home')" class="hover-cursor">Home</li>
+                <li id="about-us" onclick="onclick_navbar_item('about-us')" class="hover-cursor">About Us</li>
+                <li id="our-lands" onclick="onclick_navbar_item('our-lands')" class="hover-cursor">Our Lands</li>
+                <li id="our-suppliers" onclick="onclick_navbar_item('our-suppliers')" class="hover-cursor">Our Suppliers</li>
+                <li id="products" onclick="onclick_navbar_item('products')" class="hover-cursor">Products</li>
+                <li id="contact-us" onclick="onclick_navbar_item('contact-us')" class="hover-cursor">Contact Us</li>
+                <li id="profile-image" onclick="onclick_navbar_item('profile-image')"><img class="profile-size" src="./Assets/images/profile Image.png"></li>
```
Comment (mmasaid) [inline task_1/index.html:21]: Use kebab case for the all image, class, ids to be consistent  
```suggestion
                <li id="profile-image" onclick="onclick_navbar_item('profile-image')"><img class="profile-size" src="./Assets/images/profile-image.png"></li>
```

File: task_1/index.html [hunk 16]
```diff
@@ -1,45 +1,110 @@
...
+            </ul>
         </div>
         <div class="main">
-            <div class="hero-image"></div>
+            <div class="hero-image">
+                <div class="Card">
+                    <img class="image" src="./Assets/images/Mocha-Late.png">
+                    <p class="card-title">Mocha Late</p>
+                    <p class="description">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                    <pre class="write">1 Bag                          $69.99</pre>
```
Comment (mmasaid) [inline task_1/index.html:31]:
```suggestion
                    <pre class="item-price">1 Bag                          $69.99</pre>
```

File: task_1/index.html [hunk 17]
```diff
@@ -1,45 +1,110 @@
...
+                        <p class="description">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                    </div>
+                    <div class="coffee-item">
+                        <img src="./Assets/images/caramel-frappe.png" alt="caramel frappe">
+                        <p class="card-title">Caramel Frappe</p>
+                        <p class="description">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                    </div>
+
+                    <div class="coffee-item">
+                        <img src="./Assets/images/Mocha-Late02.png" alt="Mocha Late">
```
Comment (mmasaid) [inline task_1/index.html:54]: names convention 
```suggestion
                        <img src="./Assets/images/mocha-late02.png" alt="Mocha Late">
```

File: task_1/style.css [hunk 18]
```diff
@@ -8,6 +8,147 @@
...
+    color: #EB5757;
+    font-size: 20px;
+    font-weight: bold;
+    margin: 0;
+}
+
+.black-color {
+    font-size: 40px;
+    margin: 0;
+}
```
Comment (mmasaid) [inline task_1/style.css:84]: the color property missing depend on the class name 
```suggestion
.black-color {
    color: black;
    font-size: 40px;
    margin: 0;
}
```

File: task_1/style.css [hunk 19]
```diff
@@ -8,6 +8,147 @@
...
+.more-item {
+    background-color: white;
+    border-radius: 10px;
+    height: 200px;
+    margin:170px auto auto auto;
+    text-align: center;
+    width: 200px;
+}
+
+.Card {
```
Comment (mmasaid) [inline task_1/style.css:141]:
```suggestion
.card {
```

File: task_1/style.css [hunk 20]
```diff
@@ -8,6 +8,147 @@
...
+
+.more-item-photo{
+      margin: 30px;
+}
+
+.view-more-test{
+    margin: 0;
+}
+
+.write {
```
Comment (mmasaid) [inline task_1/style.css:105]:
```suggestion
.item-price {
```

File: task_1/style.css [hunk 21]
```diff
@@ -16,54 +157,47 @@
...
 .text-banner {
-    padding: 10px 30px;
     font-family: OpenFont;
+    padding: 10px 30px;
 }
 
 .coffee-list {
     align-items: center;
     display: flex;
+    gap: 20px;
```
Comment (mmasaid) [inline task_1/style.css:193]: use margin instead 
```suggestion
    margin :....
```

**Review by mmasaid — CHANGES_REQUESTED** [review]
Please address the comments and let me know if you have any questions

**Review by mmasaid — APPROVED** [review]

**Pull request merged**